test/e2e/virtual/syncer/virtualworkspace_test.go

```diff
@@ -196,6 +196,7 @@ func logWithTimestamp(t *testing.T, format string, args ...interface{}) {
 func TestSyncerVirtualWorkspace(t *testing.T) {
 	t.Parallel()
 	framework.Suite(t, "transparent-multi-cluster")
```

Comment:
Need to remove this line as per https://kubernetes.slack.com/archives/C021U8WSAFK/p1669750415400799